warp/Network/Wai/Handler/Warp/HTTP2/Worker.hs

```diff
@@ -236,7 +236,8 @@ worker :: Context -> S.Settings -> Application -> Responder -> T.Manager -> IO (
 worker ctx@Context{inputQ,controlQ} set app responder tm = do
     sinfo <- newStreamInfo
     tcont <- newThreadContinue
-    E.bracket (T.registerKillThread tm) T.cancel $ go sinfo tcont
+    let timeoutAction = return () -- cannot close the shared connection
```

Comment:
@kazu-yamamoto I'm not sure if this is correct, or if there is in fact some way to close the connection when the timeout is triggered.